api/Common.h

```diff
@@ -20,6 +20,7 @@ typedef enum {
   OUTPUT          = 0x1,
   INPUT_PULLUP    = 0x2,
   INPUT_PULLDOWN  = 0x3,
+  OUPUT_OPENDRAIN = 0x4
```

Comment:
@obra there's a typo here :slightly_smiling_face:  Also, please add a comma at the end so in case we need to add more (can't think about a real use case now but better being prepared) the commit would be cleaner :wink:

Reply:
🤦🏼 Well that's embarassing.  So sorry.

I've fixed the typo, added the comma and then added a space before the = on each line so everything lines up again. Force-pushed and ready for review again.